### ISSUE TITLE
No updates from ES - GeoportalGasolineras.es since some time ago

### ISSUE THREAD

**Issue by ferlanero** [xXSparkeyy/harbour-spritradar#35]
Since 1 or 2 months ago, the results showed by SpritRadar from the plugin [ES- GeoportalGasolineras.es](https://geoportalgasolineras.es/#/Inicio) doesn't match the results the web shows (Screen shots taken at the same time) Could be fixed, please. I use this SpritRadar every time I have to refuel my car. Thanks for keeping this incredible app running and up!
![Captura de pantalla de 2020-10-01 20-37-36](https://user-images.githubusercontent.com/1192395/94849988-8c0f1880-0426-11eb-9d29-b0edce7031ce.png)
![Captura_20201001_003](https://user-images.githubusercontent.com/1192395/94850043-a6e18d00-0426-11eb-956c-60307da257a9.png)

**Comment by xXSparkeyy** [comment]
Hi, I changed the datasource to use more up to date data, you might give it a try and report
https://github.com/xXSparkeyy/harbour-spritradar/blob/es_rest/RPMS/harbour-spritradar-2.11-1.armv7hl.rpm

**Comment by ferlanero** [comment]
Thanks for the update. I was testing this afternoon but now data from Spritradar seems not to fit what geoportalgasolineras.es shows. I let you screenshots:
![Captura_20201018_003](https://user-images.githubusercontent.com/1192395/96385049-9274f280-1191-11eb-9dd9-a2b447a4e4c6.png)

![Captura_20201018_002](https://user-images.githubusercontent.com/1192395/96385034-7c673200-1191-11eb-8356-8ed274ff7b74.png)

If you need more tests or something similar, please let me know. Thanks in advance.

**Comment by xXSparkeyy** [comment]
Hi, thanks for the testing. I guess the problem is that the website lists all stations in the province(?) of León, while the app only shows those which are in the given radius, maxing out at 25km, of whichever address you entered.
I've compared the data for a smaller area and they seemed to match quite well.
Maybe you could have a look at how accurate the data is the next time you fill up?

**Comment by ferlanero** [comment]
I'm testing it right now and even if I'm using the same parameters to find stations, the results differs completely between website and app (You can see it in screenshots using my postal code which is 24005. Same occurs using GPS). Results in website are fine, while not in app (even I don't know which stations are those which Spritradar shows):
![Captura_20201026_002](https://user-images.githubusercontent.com/1192395/97228419-30427000-17d7-11eb-8162-d64cb5818c7f.png)
![Captura_20201026_004](https://user-images.githubusercontent.com/1192395/97228492-4f410200-17d7-11eb-921a-457bc3e0dc95.png)
![Captura_20201026_001](https://user-images.githubusercontent.com/1192395/97228534-5a942d80-17d7-11eb-9280-5173b884a603.png)
![Captura_20201026_005](https://user-images.githubusercontent.com/1192395/97228550-5f58e180-17d7-11eb-93d2-58f60ca438a4.png)
![Captura_20201026_006](https://user-images.githubusercontent.com/1192395/97228561-641d9580-17d7-11eb-90c9-ea5c5667568b.png)

**Comment by xXSparkeyy** [comment]
Hi, I just looked into it and it seem like I got the names wrong, try out 'Gasolina 98 E5' instead of 'Gasóleo A habitual' and the other thing I noticed was that If you use the postal code the first time it works fine and the second time the adress changes slightly, so try to always insert the postal code and it should work.
Gonna fix those later

**Comment by ferlanero** [comment]
Yeah! You are right! By choosing "Gasolina 98 E5" Spritradar throws the correct result. That's great!
As soon as to the postal code I can't see more differences more than Spritradar autocompletes address with more detail one.

So I hope we can have a fixed release of one of the most used programs by me in Sailfish.

Thank you very much for developing & maintain Spritradar!!
